website/docs/r/synthetics_monitor.html.markdown

```diff
@@ -3,114 +3,115 @@ layout: "newrelic"
 page_title: "New Relic: newrelic_synthetics_monitor"
 sidebar_current: "docs-newrelic-resource-synthetics-monitor"
 description: |-
-  Create and manage a Synthetics monitor in New Relic.
+Create and manage a Synthetics monitor in New Relic.
 ---
 
 # Resource: newrelic\_synthetics\_monitor
 
-Use this resource to create, update, and delete a synthetics monitor in New Relic.
```

Comment:
Use this resource to create, update, and delete a Simple or Browser Synthetics Monitor in New Relic.